esp/esp/program/modules/apps.py

```diff
@@ -2,3 +2,9 @@
 
 class ModulesConfig(InstallConfig):
     name = 'esp.program.modules'
+
+    def ready(self):
+        super(ModulesConfig, self).ready()
+        # TODO(benkraft): add a thing to InstallConfig that imports a signals
```

Comment:
Yeah. I was going to say "huh, why did that not error?" but then I realized it's because you didn't set anything to use that AppConfig. You also need to tell Django to use that one instead of the default one by putting it in django_settings.py.